blueocean-commons/src/main/java/io/jenkins/blueocean/commons/ListsUtils.java

```diff
@@ -0,0 +1,16 @@
+package io.jenkins.blueocean.commons;
+
+import java.util.Collection;
+
+public class ListsUtils {
+    public ListsUtils() {
+        // no op
+    }
+
+    public static <T>T getFirst( Collection<T> collection, T defaultValue){
```

Comment:
```suggestion
    public static <T>T getFirst(Collection<T> collection, T defaultValue){
```

Reply:
new class so there is no formatting change..